templates/pages/login.jsx

```diff
@@ -47,6 +49,17 @@ var Login = React.createClass({
   },
   handleFormData: function(error, data) {
     console.log("inside App we see:", error, data);
+    fetch('/login', {
+      method: 'post',
+      headers: {
+        'Accept': 'application/json',
+        'Content-Type': 'application/json'
+      },
+      body: JSON.stringify({
+        uid: data.username,
+        password: data.password
+      })
+    });
```

Comment:
Also, can you make this redirect to /login/oauth/authorize with the parameters we specified before?